Automattic/jetpack
# AI Client: add play and pause icons

This PR addes the `playerPlayIcon` and `playerPauseIcon` components:

Fixes #

## Proposed changes:

* AI Client: add play and pause icons

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






### Test with Storybook
```
cd projects/js-packages/storybook/
```

```
pnpm run storybook:dev
```

* Go to AI Client icons story (http://localhost:50240/?path=/docs/js-packages-ai-client-icons--docs)
* Confirm you see the new icons there


<img width="850" alt="Screenshot 2023-08-31 at 13 10 43" src="https://github.com/Automattic/jetpack/assets/77539/9c59fcca-a68f-4b7b-b0eb-c43206c95f88">
